content/en/docs/instrumentation/java/automatic/servers.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Configuring application servers
+description: Learn how to define agent paths for Java application servers
+weight: 15
+cSpell:ignore: asadmin binsetenv binstart Glassfish Payara setenv
+---
+
+When instrumenting an application running on an application server, you've to
+define the path to the OpenTelemetry Java agent. The way to define the path
+differs from server to server.
+
+## JBoss EAP / WildFly
+
+To define the path to the Java agent, add the `javaagent` argument at the end of
+the `standalone.conf` or the `standalone.conf.bat` configuration files:
+
+```shell
+# Linux / standalone.conf
+JAVA_OPTS="$JAVA_OPTS -javaagent:/path/to/opentelemetry-javaagent.jar"
+
+# Windows / standalone.conf.bat
+set "JAVA_OPTS=%JAVA_OPTS% -javaagent:<Drive>:\path\to\opentelemetry-javaagent.jar"
+```
+
+## Jetty
+
+To define the path to the Java agent, use the `-javaagent` argument:
+
+```shell
+java -javaagent:/path/to/opentelemetry-javaagent.jar -jar start.jar
+```
+
+If you use the `jetty.sh` file to start Jetty, add the following line to the
+\<jetty_home\>/bin/jetty.sh file:
+
+```shell
+JAVA_OPTIONS="${JAVA_OPTIONS} -javaagent:/path/to/opentelemetry-javaagent.jar"
+```
+
+If you use the start.ini file to define JVM arguments, add the `javaagent`
+argument after the `--exec` option:
+
+```shell
+#===========================================================
+# Sample Jetty start.ini file
+#-----------------------------------------------------------
+--exec
+-javaagent:/path/to/opentelemetry-javaagent.jar
+```
+
+## Glassfish / Payara
+
+Add the path to the Java agent using the `asadmin` tool:
+
+```shell
+# Linux
+<server_install_dir>/bin/asadmin create-jvm-options "-javaagent\:/path/to/opentelemetry-javaagent.jar"
+
+# Windows
+<server_install_dir>\bin\asadmin.bat create-jvm-options '-javaagent\:<Drive>\:\\path\\to\\opentelemetry-javaagent.jar'
+```
```

Comment:
~~~suggestion
{{< tabpane text=true >}} {{% tab Linux %}}

```sh
<server_install_dir>/bin/asadmin create-jvm-options "-javaagent\:/path/to/opentelemetry-javaagent.jar"
```

{{% /tab %}} {{% tab Windows %}}

```powershell
<server_install_dir>\bin\asadmin.bat create-jvm-options '-javaagent\:<Drive>\:\\path\\to\\opentelemetry-javaagent.jar'
```

{{% /tab %}} {{< /tabpane >}}
~~~

**NOTE** the use of the `powershell` lang for Windows.